Source/WebCore/css/makeprop.pl

```diff
@@ -1558,10 +1570,18 @@ sub generateValueSetter {
       $setterContent .= "    ";
     }
     $setterContent .= $indent . "    " . generateSetValueStatement($name, $convertedValue) . ";\n";
+    $didUseValue = 1;
   }
   if (exists($propertiesWithStyleBuilderOptions{$name}{"fast-path-inherited"})) {
     $setterContent .= $indent . "    builderState.style().setDisallowsFastPathInheritance();\n";
   }
+  if (exists $propertiesWithStyleBuilderOptions{$name}{"is-appearance-disabling-property"}) {
+    if (!($didUseValue or $didCallSetValue)) {
+      $setterContent .= $indent . "    UNUSED_PARAM(value);\n";
```

Comment:
Why is this necessary for `is-appearance-disabling-property` and not the others?

Reply:
because all the others actually use `value`

Reply:
But this code is additive? I'm not sure I see the scenario where `value` is unused.

Can you share an example?